.github/workflows/update_llm_perf_intel_pytorch.yml

```diff
@@ -0,0 +1,56 @@
+name: Update LLM Perf Benchmarks - Intel PyTorch
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
this has been launching the PR in the past days, hope it's intentional and didn't pollute the datasets 😅

Reply:
No, it's fine and by design. There is also a 6h timeout on the runners so having multiple scheduled runs allowed me to actually run the full benchmark